app/main/views/services.py

```diff
@@ -51,14 +51,28 @@ def view(service_id):
         flash({'api_error': service_id}, 'error')
         return redirect(url_for('.index'))
 
+    most_recent_audit_events = data_api_client.find_audit_events(
```

Comment:
Can we check the service status here and not bother fetching any audit events at all if the service is 'published' (as 99% of services will be).  Searching the audit table feels like a reasonably heavy query on the API, and even though I think we have a partial index to support this one, it would be better to not make the call if we already know we're not going to need any audit information.

Reply:
Yeah, good call. I might have to stick up a permanent post it note to myself saying 'QUERYING AUDIT EVENTS = EXPENSIVE' as I do forget sometimes...